docs/api/core.md

```diff
@@ -5,79 +5,89 @@
 
 .. autoclass:: AECEnv
 
-    .. py:attribute:: agents
+```
+
+## Attributes
+
+
+```{eval-rst}
+
+.. autoattribute:: AECEnv.agents
 
-        A list of the names of all current agents, typically integers. These may be changed as an environment progresses (i.e. agents can be added or removed).
+    A list of the names of all current agents, typically integers. These may be changed as an environment progresses (i.e. agents can be added or removed).
 
-        :type: list[AgentID]
+    :type: List[AgentID]
 
-    .. py:attribute:: num_agents
+.. autoattribute:: AECEnv.num_agents
 
-        The length of the agents list.
+    The length of the agents list.
 
-        :type: int
+    :type: int
```

Comment:
Don't need these

Reply:
ok

Reply:
Should I keep `List[AgentID]` which is more informative than `List[str]`?